static/app/components/replays/table/replayTableColumns.tsx

```diff
@@ -522,18 +523,40 @@ export const ReplaySessionColumn: ReplayTableColumn = {
 
     const referrer = getRouteStringFromRoutes(routes);
     const eventView = EventView.fromLocation(location);
+
+    const {statsPeriod, ...eventViewQuery} = eventView.generateQueryStringObject();
+
+    let query = {
```

Comment:
[logaf](https://blog.danlew.net/2020/04/15/the-logaf-scale/) low: we use `query` in a lot of places, we could be a bit more specific and name this `detailsTabQuery`